charts/mirrorbits-parent/templates/_helpers.tpl

```diff
@@ -13,13 +14,14 @@ Expand the name of the chart.
 
 {{/*
 Create chart name and version as used by the chart label.
+Must be 63 chars. maximum.
```

Comment:
```suggestion
```
Can be greater than 63 chars as it will be truncated to 63 anyway?

Reply:
The comment is about the result of the template function (and its usage), not about the values that could be used as part of the function.

The helper template is not a user-facing artefact in helm chart.